src/pages/Auth/auth.css

```diff
@@ -0,0 +1,30 @@
+.signup-card {
+  margin: 1rem auto;
+}
+.login-card {
+  margin: 4rem auto;
+}
+.signup,
+.login {
+  max-width: 28rem;
+  margin: auto;
+  padding: 1rem;
+  background-color: var(--off-white);
+  box-shadow: rgba(0, 0, 0, 0.07) 0px 1px 1px, rgba(0, 0, 0, 0.07) 0px 2px 2px,
```

Comment:
try storing the big values inside a variable, and use it here

Reply:
Yeah Will do that